### ISSUE TITLE
bibox SSL3_GET_SERVER_CERTIFICATE:certificate verify failed

### ISSUE THREAD

**Issue by flaeshi** [ccxt/ccxt#2193]
Hi,

I am getting a SSL error for the bibox exchange

- OS: Linux Mint
- Programming Language version: Python 2.7.6
- CCXT version: 1.11.53
- Exchange: bibox
- Method: fetch_order_book()
- Based in Germany

Is there a solution to this?

Input:
```
import ccxt
bibox = ccxt.bibox()
bibox.apiKey = 'ABC'
bibox.secret = 'DEF'
bibox.verbose = True
print bibox.fetch_order_book('CPC/ETH')
```

Output:

```/usr/bin/python2.7 /home/josef/PycharmProjects/Test.py
('\nRequest:', 'GET', 'https://api.bibox.com/v1/mdata?cmd=marketAll', {'Content-Type': 'application/json', 'Accept-Encoding': 'gzip, deflate', 'User-Agent': 'python-requests/2.18.4'}, None)
/usr/local/lib/python2.7/dist-packages/urllib3/util/ssl_.py:339: SNIMissingWarning: An HTTPS request has been made, but the SNI (Subject Name Indication) extension to TLS is not available on this platform. This may cause the server to present an incorrect TLS certificate, which can cause validation failures. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  SNIMissingWarning
/usr/local/lib/python2.7/dist-packages/urllib3/util/ssl_.py:137: InsecurePlatformWarning: A true SSLContext object is not available. This prevents urllib3 from configuring SSL appropriately and may cause certain SSL connections to fail. You can upgrade to a newer version of Python to solve this. For more information, see https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecurePlatformWarning
Traceback (most recent call last):
  File "/home/josef/PycharmProjects/Test.py", line 11, in <module>
    print bibox.fetch_order_book('ETH/CPC')
  File "/usr/local/lib/python2.7/dist-packages/ccxt/bibox.py", line 225, in fetch_order_book
    self.load_markets()
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 860, in load_markets
    markets = self.fetch_markets()
  File "/usr/local/lib/python2.7/dist-packages/ccxt/bibox.py", line 95, in fetch_markets
    }, params))
  File "/usr/local/lib/python2.7/dist-packages/ccxt/bibox.py", line 531, in request
    response = self.fetch2(path, api, method, params, headers, body)
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 289, in fetch2
    return self.fetch(request['url'], request['method'], request['headers'], request['body'])
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 367, in fetch
    self.raise_error(ExchangeError, url, method, e, self.last_http_response)
  File "/usr/local/lib/python2.7/dist-packages/ccxt/base/exchange.py", line 274, in raise_error
    raise exception_type(output)
ccxt.base.errors.ExchangeError: bibox https://api.bibox.com/v1/mdata?cmd=marketAll GET HTTPSConnectionPool(host='api.bibox.com', port=443): Max retries exceeded with url: /v1/mdata?cmd=marketAll (Caused by SSLError(SSLError(1, '_ssl.c:510: error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed'),))

Process finished with exit code 1
```

**Comment by flaeshi** [comment]
I fixed the error by updating to python 2.7.14

**Comment by kroitor** [comment]
@flaeshi thx for reporting back!
